tests/common/bats/env.bash

```diff
@@ -106,6 +106,9 @@ env.init() {
     yq.set 'sc' '.alerts.opsGenieHeartbeat.name' '"example-heartbeat-name"'
   fi
 
+  yq.set 'secrets' '["alerts"]["opsGenie"]["apiKey"]' '"ZXhhbXBsZS1hcGlrZXkx"'
+  yq.set 'secrets' '["alerts"]["slack"]["apiUrl"]' '"https://slack.example/channel/ZXhhbXBsZS1jaGFubmVs"'
+
```

Comment:
Also looks unrelated

Reply:
I think this was needed by the tests because otherwise it complains about "set-me". Do you remember @aarnq ?

Reply:
Feels like it should have failed before this as well then? It's not a biggie though but curious.

Reply:
It is because the migration lib (which hasn't been covered in tests before) is stricter than bin scripts in that it doesn't like somelongsecrets.